packages/peer-discovery-mdns/README.md

```diff
@@ -1,30 +1,17 @@
-# @libp2p/mdns <!-- omit in toc -->
-
 [![libp2p.io](https://img.shields.io/badge/project-libp2p-yellow.svg?style=flat-square)](http://libp2p.io/)
 [![Discuss](https://img.shields.io/discourse/https/discuss.libp2p.io/posts.svg?style=flat-square)](https://discuss.libp2p.io)
 [![codecov](https://img.shields.io/codecov/c/github/libp2p/js-libp2p.svg?style=flat-square)](https://codecov.io/gh/libp2p/js-libp2p)
 [![CI](https://img.shields.io/github/actions/workflow/status/libp2p/js-libp2p/main.yml?branch=master\&style=flat-square)](https://github.com/libp2p/js-libp2p/actions/workflows/main.yml?query=branch%3Amaster)
 
 > Node.js libp2p mDNS discovery implementation for peer discovery
 
-## Table of contents <!-- omit in toc -->
+# About
 
-- [Install](#install)
-- [Usage](#usage)
-- [MDNS messages](#mdns-messages)
-- [API Docs](#api-docs)
-- [License](#license)
-- [Contribution](#contribution)
+A peer discover mechanism that uses [mDNS](https://datatracker.ietf.org/doc/html/rfc6762) to discover peers on the local network.
 
-## Install
+## Example
 
-```console
-$ npm i @libp2p/mdns
-```
-
-## Usage
-
-```Typescript
+```ts
```

Comment:
You've chosen to keep the Typescript example here (in the README) but moved it to the `@packageDocumentation` in other instances, is this intentional?